### ISSUE TITLE
[0.72.3, Android] Error: Duplicate resources

### ISSUE THREAD

**Issue by 7dp** [facebook/react-native#40724]
### Description

Generating apk fails, due to the error:
```bash
> Task :app:mergeReleaseResources FAILED
ERROR:[drawable-mdpi-v4/node_modules_reactnative_libraries_newappscreen_components_logo] /Users/wicakradityo/Documents/my_projects/react_native_projects/combat/RN723/android/app/src/main/res/drawable-mdpi/node_modules_reactnative_libraries_newappscreen_components_logo.png [drawable-mdpi-v4/node_modules_reactnative_libraries_newappscreen_components_logo] /Users/wicakradityo/Documents/my_projects/react_native_projects/combat/RN723/android/app/build/generated/res/createBundleReleaseJsAndAssets/drawable-mdpi/node_modules_reactnative_libraries_newappscreen_components_logo.png: Resource and asset merger: Duplicate resources
w: Detected multiple Kotlin daemon sessions at build/kotlin/sessions

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:mergeReleaseResources'.
> [drawable-mdpi-v4/node_modules_reactnative_libraries_newappscreen_components_logo] /Users/wicakradityo/Documents/my_projects/react_native_projects/combat/RN723/android/app/src/main/res/drawable-mdpi/node_modules_reactnative_libraries_newappscreen_components_logo.png    [drawable-mdpi-v4/node_modules_reactnative_libraries_newappscreen_components_logo] /Users/wicakradityo/Documents/my_projects/react_native_projects/combat/RN723/android/app/build/generated/res/createBundleReleaseJsAndAssets/drawable-mdpi/node_modules_reactnative_libraries_newappscreen_components_logo.png: Error: Duplicate resources

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

See https://docs.gradle.org/8.0.1/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 12m 6s
```

### React Native Version

0.72.3

### Output of `npx react-native info`

```bash
System:
  OS: macOS 12.6
  CPU: (4) x64 Intel(R) Core(TM) i5-5257U CPU @ 2.70GHz
  Memory: 74.50 MB / 8.00 GB
  Shell:
    version: 5.8.1
    path: /bin/zsh
Binaries:
  Node:
    version: 18.17.1
    path: ~/.nvm/versions/node/v18.17.1/bin/node
  Yarn:
    version: 1.22.19
    path: ~/.nvm/versions/node/v18.17.1/bin/yarn
  npm:
    version: 9.6.7
    path: ~/.nvm/versions/node/v18.17.1/bin/npm
  Watchman:
    version: 2023.08.28.00
    path: /usr/local/bin/watchman
Managers:
  CocoaPods:
    version: 1.11.3
    path: /usr/local/bin/pod
SDKs:
  iOS SDK:
    Platforms:
      - DriverKit 22.1
      - iOS 16.1
      - macOS 13.0
      - tvOS 16.1
      - watchOS 9.1
  Android SDK:
    API Levels:
      - "28"
      - "29"
      - "30"
      - "31"
      - "32"
      - "33"
    Build Tools:
      - 28.0.3
      - 29.0.2
      - 29.0.3
      - 30.0.2
      - 30.0.3
      - 31.0.0
      - 33.0.0
    System Images:
      - android-33 | Google APIs Intel x86_64 Atom
    Android NDK: Not Found
IDEs:
  Android Studio: 2022.1 AI-221.6008.13.2211.9619390
  Xcode:
    version: 14.1/14B47b
    path: /usr/bin/xcodebuild
Languages:
  Java:
    version: 11.0.20
    path: /usr/bin/javac
  Ruby:
    version: 2.7.5
    path: /Users/wicakradityo/.rbenv/shims/ruby
npmPackages:
  "@react-native-community/cli": Not Found
  react:
    installed: 18.2.0
    wanted: 18.2.0
  react-native:
    installed: 0.72.3
    wanted: 0.72.3
  react-native-macos: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: true
  newArchEnabled: false
iOS:
  hermesEnabled: true
  newArchEnabled: false
```

### Steps to reproduce

1. init new bare RN project (version 0.72.3)
2. add & configure release keystore
3. build apk by running `cd android; ./gradlew assembleRelease; cd ..`
4. apk generated successfully. But when i install it there is no React Native logo appeared. The text "Welcome to React Native" is appear but not the logo.
5. I think i forgot to bundle the JS. So i create a folder named `assets` in `android/app/src/main/`. Then i try to bundle it by running:
```
react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res/
```
6. Bundle success.
7. i try to rebuild the apk by running `cd android; ./gradlew assembleRelease; cd ..` again
8. And now build apk fails with the error mentioned above

Any help would be much appreciated. Thanks😊.

### Snack, screenshot, or link to a repository

https://github.com/7dp/RNReproDuplicateResource

**Comment by cortinico** [comment]
You have this file `node_modules_reactnative_libraries_newappscreen_components_logo.png` twice in your project.

This is unrelated to React Native but it's an Android failure

**Comment by bihongbin3027** [comment]
@7dp hi，Have you resolved it? I encountered the same problem.

**Comment by 7dp** [comment]
Hi @bihongbin3027 ,
Sorry, i ended up ignoring it.
And the issue didn't appear on my brand new bare RN project.

**Comment by itzfeltrin** [comment]
I spent a long time on this and what finally fixed it was removing the "'react-native/Libraries/NewAppScreen'" import that comes in the base template in App.tsx(js)

**Comment by ansarikhurshid786** [comment]
> I got the same issue currently. I solved by removing "--assets-dest android/app/src/main/res" in the "npx react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res" command in the yml file. "npx react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle" simply works for me.

I was facing same issue. Before building for release, i was running npx react-native bundle ...

what works for me is: 
first i have removed copied resource from android directory bye giving command (macos)
`rm -rf app/src/main/res/drawable-* app/src/main/res/raw`

then u can release building using command 
`npx react-native build-android --mode=release`
or android stuido 

no need npx react-native bundle (internally this command)
if you run manually then remove **--bundle-output android/app/src/main/assets/index.android.bundle** as mentioned by @ChanHongMing 

i can now release build 

my case was 
i upgrade from react-native 0.70 to 0.71

**Comment by JVGS1111** [comment]
I was upgrading my RN project from 0.70 to 0.71 like @ansarikhurshid786, but my solution was different.
I had to remove the following line from app/build.gradle:

`apply from: "../../node_modules/react-native/react.gradle"`

I realized this by comparing a blank project with mine.
That line probably should have been removed in a previous upgrade, but someone must have forgotten.
After removing it, my project worked fine.
